Sphinxsetup.sh

```diff
@@ -7,27 +7,41 @@ if [ "$UID" -gt 0 ]; then
      exit 1
 fi
 
-DISTRIBUTION_CODENAME=$(lsb_release -i -s)
-if [ ${DISTRIBUTION_CODENAME} == 'Ubuntu' ]; then
-  add-apt-repository universe
+DISTRIBUTION_ID=$(lsb_release -i -s)
+if [ ${DISTRIBUTION_ID} == 'Ubuntu' ]; then
+  DISTRIBUTION_CODENAME=$(lsb_release -c -s)
+  if [ ${DISTRIBUTION_CODENAME} == 'focal' ] || [ ${DISTRIBUTION_CODENAME} == 'bionic' ]; then
+    add-apt-repository universe
+  else
+    echo "Sorry, this script is meant for Ubuntu 18.04 or 20.04"
+    exit 1
+  fi
 fi
+
 apt-get -y update
-apt-get install -y unzip git imagemagick curl wget make python3 python-is-python3
+apt-get install -y unzip git imagemagick curl wget make python3
+
+# Install packages release specific
+if [ ${DISTRIBUTION_CODENAME} == 'bionic' ]; then
+  apt-get install -y python3-distutils
+elif [ ${DISTRIBUTION_CODENAME} == 'focal' ]; then
+  apt-get install -y python-is-python3
+fi
 
 # Get pip through the official website to get the lastest release
 curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py && rm -f get-pip.py
 
 # Install sphinx
-pip install -U sphinx==1.8.3 || pip3 install -U sphinx==1.8.3
+pip install --upgrade sphinx==1.8.3 || pip3 install --upgrade sphinx==1.8.3
 
 # Install sphinx theme from ArduPilot repository
-pip install git+https://github.com/ArduPilot/sphinx_rtd_theme.git -UI ||  pip3 install git+https://github.com/ArduPilot/sphinx_rtd_theme.git -UI
+pip install git+https://github.com/ArduPilot/sphinx_rtd_theme.git --upgrade ||  pip3 install git+https://github.com/ArduPilot/sphinx_rtd_theme.git --upgrade
```

Comment:
This won't do it, it got a pr that correct that

Reply:
This certainly fixes the problem in front of us - the `-I` was causing Sphinx to be updated to a version that isn't compatible with our configuration files.